### PR TITLE
Added/updated Behaviour Twin KIT logos

### PR DESCRIPTION
**Pull request by afederl**
## Description

Added/updated Behaviour Twin KIT logos for the folowing KITS:
 - Remaining useful Life KIT
 -  (RuL) Health Indicators KIT (HI)
 - Model Based Development and Data Processing KIT (MDP).

The header logos and the gallery logos were added/updated.

ToDo: Model Based Development and Data Processing KIT should be included into the KIT gallery.

### CODE REVIEW

**Review by maximilianong — APPROVED** [review]

**Comment by maximilianong** [comment]
@afederl you need a second approval by a committer. Maybe @danielmiehle can take this.

**Review by danielmiehle — APPROVED** [review]

**Comment by danielmiehle** [comment]
@afederl is this PR still valid? If yes, can you pleas resolve the conflicts, so that we can merge it. Thanks!

**Comment by afederl** [comment]
> @afederl is this PR still valid? If yes, can you pleas resolve the conflicts, so that we can merge it. Thanks!

@danielmiehle: Yes, it's still valid. I've resolved the conflicts. The markdown liniting warnings are from other KITs.

**Review by danielmiehle — APPROVED** [review]

**Review by maximilianong — APPROVED** [review]

**Pull request merged**
